### ISSUE TITLE
Patch for /Vshctrl/usbdevice.c

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#507]
```
use config file for usbdevice module name
```

Original issue reported on code.google.com by `crah...@hotmail.com` on 11 Sep 2012 at 11:25

Attachments:
- [usbdevice.c.patch](https://storage.googleapis.com/google-code-attachments/procfw/issue-507/comment-0/usbdevice.c.patch)

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Indeed, the code is inconsistent because it is defined in the config file but 
not used in the module.

Thank you for the patch
```

Original comment by `devnonam...@gmail.com` on 28 Aug 2013 at 10:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 41421afce881   
```

Original comment by `devnonam...@gmail.com` on 25 Sep 2013 at 1:50
- Changed state: **Fixed**
